src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
+
+  const sourceArray = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  sourceArray.forEach((elem) => {
+    const startValueIndex = elem.indexOf(':');
+
+    if (startValueIndex !== -1) {
+      const propertyName = elem.slice(0, startValueIndex).trim();
+      const value = elem.slice(startValueIndex + 1).trim();
+
+      if (propertyName && value) {
+        convertedObj[propertyName] = value;
+      }
+    }
+  });
```

Comment:
[CODE KNOWLEDGE] Instead of using `forEach` and then modifying `convertedObj` inside the loop, you can use `reduce` method. It's a more functional approach and it doesn't mutate the existing object.